src/main/resources/db/migration/V20210608__CCD-1249_add_index_hearingVenue.sql

```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX uidx_case_data_hearing_venue ON public.case_data USING btree (btrim(upper((data #>> '{hearingVenue}'::text[]))));
```

Comment:
we should create this if not already existing.

Reply:
Updated the PR